patches/components-omnibox-browser-omnibox_view.cc.patch

```diff
@@ -0,0 +1,13 @@
+diff --git a/components/omnibox/browser/omnibox_view.cc b/components/omnibox/browser/omnibox_view.cc
+index f72a61d6917ac72d1138f7af56ff6ba6aa1dd43b..a029efade003119cf63f03169e712d947582d1d5 100644
+--- a/components/omnibox/browser/omnibox_view.cc
++++ b/components/omnibox/browser/omnibox_view.cc
+@@ -364,7 +364,7 @@ OmniboxView::StateChanges OmniboxView::GetStateChanges(const State& before,
+ }
+ 
+ OmniboxView::OmniboxView(std::unique_ptr<OmniboxClient> client)
+-    : controller_(std::make_unique<OmniboxController>(
```

Comment:
seems like it would be better to override `StartAutocomplete` in `OmniboxController` now and not subclass so we don't have to patch here?

Reply:
Addressed in https://github.com/brave/brave-core/pull/19398/commits/8aa010353879e28422b55a974a7e7a3451da4f71